### PR TITLE
ref/update_setimageurl

### PR DESCRIPTION
**Pull request by alhiwatan**
Update to use setImageUrl from MAX SDK.   Verified to work.

### CODE REVIEW

**Review by thomasmso — COMMENTED** [review]
You have to update the AppLovin MAX SDK dependencies to v11.11.0 first, otherwise `AppLovinSdkUtils.setImageUrl(...)` is not available in the plugin's current state. You can check out one of my other commits for updating AppLovin SDK dependency

**Comment by alhiwatan** [comment]
Just opened a PR to update the AppLovin MAX SDK dependencies to v11.11.0.    What is the naming convention of a branch for the dependency update?

File: applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXNativeAdView.java [hunk 1]
```diff
@@ -566,30 +565,4 @@ private void maybeDestroyCurrentAd()
 
         clickableViews.clear();
     }
-
-    private void setImageUrl(final String imageUrl, final ImageView imageView)
```
Comment (thomasmso) [inline applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXNativeAdView.java:570]: Did you auto-optimize imports as well?

**Review by thomasmso — APPROVED** [review]

File: applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXNativeAdView.java (hunk 1 above)
Comment (alhiwatan) [reply applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXNativeAdView.java:570]: Oh, my bad.  Forgot to run optimize import.  Will push this now.

**Pull request merged**